src/components/node-list/node-list-items.js

```diff
@@ -330,3 +350,333 @@ export const getFilteredItems = createSelector(
     };
   }
 );
+
+/**
+ * returns the corresponding filtered parent moduar pipelines
+ * for each filtered node
+ */
+export const getFilteredNodeModularPipelines = createSelector(
+  [
+    getFilteredNodeItems,
+    getFilteredModularPipelineItems,
+    (state) => state.modularPipelines,
+    (state) => state.nodeTypeIDs,
+  ],
+  (
+    filteredNodeItems,
+    filteredModularPipelines,
+    modularPipelines,
+    nodeTypeIDs
+  ) => {
+    const filteredNodeMoudularPipelines = [];
+    const filteredMP = filteredModularPipelines.modularPipeline;
```

Comment:
yes it is - this is an unfortunate situation due to the previous setup where `filteredModularPipelines` returns the pipelines array under an additional `modularPipeline` key - here I am trying to introduce non-breaking code given that Liam's filter work would also rely on this selector, hence I kept it in this rather than modifying the previous selector.